src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,38 @@
 
 class Animal {
   // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
   // write your code here
+  constructor() {
+    super();
```

Comment:
[FUNCTION NAMING] - The constructor of the Herbivore class should take `name` as a parameter and pass it to the superclass constructor using `super(name)`. As per the task description, all instances of the `Animal` class must have a `name` property, so the constructor should be designed to take the name as an argument.